XenAdmin/Wizards/PatchingWizard/PatchingWizard.cs

```diff
@@ -188,6 +188,7 @@ protected override void UpdateWizardContent(XenTabPage senderPage)
 
                     PatchingWizard_PatchingPage.Patch = PatchingWizard_UploadPage.Patch;
                 }
+                PatchingWizard_PrecheckPage.Pool_update = PatchingWizard_UploadPage.PoolUpdate;

```

Comment:
Would it be possible to rename the PatchingWizard_PrecheckPage.Pool_update property to PoolUpdate, to match the coresponding properties on the other pages?